rag/llm/ocr_model.py

```
@@ -98,6 +99,66 @@ def parse_pdf(self, filepath: str, binary=None, callback=None, parse_method: str
 
 
+class MonkeyOCROcrModel(Base, MonkeyOCRParser):
+    _FACTORY_NAME = "MonkeyOCR"
+
+    def __init__(self, key: str | dict, model_name: str, **kwargs):
+        Base.__init__(self, key, model_name, **kwargs)
+        raw_config = {}
+        if key:
+            try:
+                raw_config = json.loads(key)
+            except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)